CFUValidator/CFUValidator.cs

```diff
@@ -115,13 +115,13 @@ private List<string> GetVersionToCheck(List<XenServerVersion> xenServerVersions)
             SetupMocks(xenServerPatches, xenServerVersions);
 
             Status = "Determining XenCenter update required...";
-            var xcupdateAlert = XenAdmin.Core.Updates.NewXenCenterUpdateAlert(xenCenterVersions, new Version(ServerVersion));
+            var xcupdateAlerts = XenAdmin.Core.Updates.NewXenCenterUpdateAlerts(xenCenterVersions, new Version(ServerVersion));
 
             Status = "Determining XenServer update required...";
-            var updateAlert = XenAdmin.Core.Updates.NewXenServerVersionAlert(xenServerVersions);
+            var updateAlerts = XenAdmin.Core.Updates.NewXenServerVersionAlerts(xenServerVersions);
 
             Status = "Determining patches required...";
-            var patchAlerts = XenAdmin.Core.Updates.NewXenServerPatchAlerts(xenServerVersions, xenServerPatches).Where(alert => !alert.CanIgnore).ToList();
+            var patchAlerts = XenAdmin.Core.Updates.NewXenServerPatchAlerts(xenServerVersions, xenServerPatches).ToList();

```

Comment:
As discussed, we should keep `alert.CanIgnore` here and see why it always returns true for the mock connection.